src/test/java/com/easyrest/tests/BaseTest.java

```diff
@@ -61,7 +62,7 @@ public void getScreenShotPath(String testCaseName) throws IOException {
         FileUtils.copyFile(source, new File(destinationFile));
     }
 
-    @AfterTest
+    @AfterClass
```

Comment:
> You can not change this annotation because the driver is initialized in the BeforeTest method, so it should be closed in the AfterTest. These operations should be consistent.

can I change BeforeClass and AfterClass?

Reply:
> You can not change this annotation because the driver is initialized in the BeforeTest method, so it should be closed in the AfterTest. These operations should be consistent.

fix it